interface/main/tabs/css/menu.css

```diff
@@ -149,6 +149,7 @@ and open the template in the editor.
     padding-left: 20px;  
     padding-bottom : 0px; 
     font-weight : 600;
+    background-color: #fff; // changes drop-down sub-menu to white
 }   
```

Comment:
@KoniKodes seems to do the trick will test this in a bit.

Reply:
Thank you. It worked in the Inspector...